contracts/modules/LoanClosings.sol

```diff
@@ -466,6 +503,7 @@ contract LoanClosings is LoanClosingsEvents, VaultController, InterestUser, Swap
 
         Loan storage loanLocal = loans[loanId];
         LoanParams storage loanParamsLocal = loanParams[loanLocal.loanParamsId];
+        //TODO should we skip this check if invoked from rollover ?
```

Comment:
it will probably cause the close to fail. did you test it?
the answer is: we should skip this when called from rollover.
david already extracted the check to the public depositWithCollateral function, I think. In that case, you would just need to merge his changes and it should work. However, his PR hasn't been reviewed, yet. So,  better wait a bit.